src/utils/domainMetadata.ts

```diff
@@ -236,5 +236,16 @@ export const DomainMetadataConfig: EnvironmentMetadataConfigType = {
       blockExplorerUrl: "https://polygonscan.com",
       renderName: "Polygon",
     },
+    8: {
+      icon: "https://framerusercontent.com/images/BapvKsJyYZUbcVcnjY2XmyaA.svg",
```

Comment:
we already have the icon, so you can use this `https://scan.buildwithsygma.com/assets/icons/b3-sepolia.svg`